e2e/pod.go

```diff
@@ -395,3 +395,22 @@ func getKernelVersionFromDaemonset(f *framework.Framework, ns, dsn, cn string) (
 
 	return kernelRelease, nil
 }
+
+func recreateCSIPods(f *framework.Framework, podLabel, daemonsetName, deploymentName string) error {
```

Comment:
Comments for the parameters might be useful. Also a description containing the behaviour (what is expected if the pods already/partially exist?) would be welcome.

Reply:
Done. it issue `delete` operation on the POD and rest is same as what CO does today.